helpers/general/getClassNames.ts

```diff
@@ -0,0 +1,16 @@
+import { TypeClassNames } from '@/types/index'
+
+type TypegetClassNamesProps = TypeClassNames
```

Comment:
should follow camel case pattern
```suggestion
type TypeGetClassNamesProps = TypeClassNames
```